hasher-matcher-actioner/hmalib/models.py

```diff
@@ -511,16 +474,20 @@ class BankedSignal:
     bank_source: str
     classifications: t.List[str] = field(default_factory=list)
 
-    def to_dict(self) -> dict:
-        return {
-            "BankedContentId": self.banked_content_id,
-            "BankId": self.bank_id,
-            "BankSource": self.bank_source,
-            "Classifications": self.classifications,
-        }
 
-    @classmethod
-    def from_dict(cls, d: dict) -> "BankedSignal":
-        return cls(
-            d["BankedContentId"], d["BankId"], d["BankSource"], d["Classifications"]
-        )
+@dataclass
+class MatchMessage(HasAWSSerialization):
```

Comment:
I had to move the match message dataclass to be after banked signal so that the typing of the matching banked signals field could be a direct reference.